test/assigned_fulfillment_order_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+class AssignedFulFillmentOrderTest < Test::Unit::TestCase
+  context "AssignedFulfillmentOrder" do
+    context "#all" do
+      should "be able to list assigned fulfillment orders by assigned_status" do
+        fo_fixture = load_fixture('assigned_fulfillment_orders')
+        fake 'assigned_fulfillment_orders.json?assigned_status=cancellation_requested', method: :get,
```

Comment:
I know it may not be feasible to always keep these things in sync, but it's somewhat confusing that the fixtures have `request_status: unsubmitted`, but will still be returned as part of this request, which passes in `assigned_status: 'cancellation_requested'`.

From here, it looks like the FOs returned should only be `in_progress/cancellation_accepted`
https://github.com/Shopify/shopify/blob/4ac84b757476157526b88347af3c970fe9e5cbcb/components/fulfillments/app/models/fulfillments/fulfillment_order_assignment_status.rb#L18-L22

Reply:
Updated the `assigned_fulfillment_orders` fixture data to be logically consistent.